screener.config.js

```diff
@@ -7,5 +7,5 @@ module.exports = {
   storybookConfigDir: ".storybook",
   storybookStaticDir: "./.storybook/assets",
   failureExitCode: 0, // Don't fail CI builds
-  includeRules: [/Automation Tests\/Screener/, /^Stylers/, /^Button: Basic/],
```

Comment:
We aren't using Automation Tests folder anymore?

Reply:
We are, but this condition would only catch stories in a `Screener` subfolder of `Automation Tests`.  Since I only have one test story (called `Screener` in `Automation Tests`) I don't want the extra layer.  And then though, well, any stories with `Screener` in the name, or in a `Screener` subfolder would be reasonable to snapshot.